packages/models-library/tests/test__models_fit_schemas.py

```diff
@@ -28,7 +28,7 @@ def test_generated_schema_same_as_original(
     # TODO: create instead a fixture that returns a Callable and do these checks
     # on separate test_* files that follow the same package submodule's hierarchy
     #
-    generated_schema = json.loads(pydantic_model.schema_json(indent=2))
+    generated_schema = json_loads(pydantic_model.model_json_schema(), indent=2)
```

Comment:
so question:
above you have stuff like `json_dumps(X.model_json_schema())` and here you do json_loads without json_dumps.
So is this json_dumps really needed?

Reply:
this is indeed a mistake